lib/iris/fileformats/pp_load_rules.py

```diff
@@ -1155,6 +1155,27 @@ def _all_other_rules(f):
             )
         )
 
+    if (
+        f.bdx != 0.0
+        and f.bdx != f.bmdi
+        and len(f.lbcode) != 5
+        and f.lbcode[0] == 9
+    ):
+        dim_coords_and_dims.append(
+            (
+                DimCoord.from_regular(
+                    f.bzx,
+                    f.bdx,
+                    f.lbnpt,
+                    standard_name="projection_x_coordinate",
```

Comment:
The [CF standard name table](http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html) states that "projection_x_coordinate" refers to

> "x" indicates a vector component along the grid x-axis, when this is not true longitude, positive with increasing x. Projection coordinates are distances in the x- and y-directions on a plane onto which the surface of the Earth has been projected according to a map projection. The relationship between the projection coordinates and latitude and longitude is described by the grid_mapping.

So that sounds about right to me!